### PR TITLE
Keyboard navigation

### LINKED ISSUE #218: Keyboard support for AssetSearch
- Cycle results with arrow keys
- Apply selection with `enter` key

### PR DESCRIPTION
**Pull request by follan**
Fixes #218

### CODE REVIEW

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-228.gearboxjs.preview.cogniteapp.com

File: src/components/common/Search/Search.tsx [hunk 1]
```diff
@@ -202,6 +209,43 @@ export class Search extends React.Component<SearchProps, SearchState> {
...
+    if (e.keyCode === 38 && cursor && cursor > 0) {
+      this.setState({ cursor: cursor - 1 });
+      return;
+    }
+
+    // Arrow down
+    if (e.keyCode === 40) {
+      if (cursor === undefined) {
+        this.setState({ cursor: 0 });
+      } else if (cursor < liveSearchResults.length - 1) {
```
Comment (f1cognite) [inline src/components/common/Search/Search.tsx]: Not loop around the results?

File: src/components/common/Search/Search.tsx [hunk 2]
```diff
@@ -202,6 +209,43 @@ export class Search extends React.Component<SearchProps, SearchState> {
...
+      e.preventDefault();
+    }
+
+    if (!liveSearch || !liveSearchShow) {
+      return;
+    }
+
+    // Arrow up
+    if (e.keyCode === 38 && cursor && cursor > 0) {
+      this.setState({ cursor: cursor - 1 });
```
Comment (f1cognite) [inline src/components/common/Search/Search.tsx]: Not handle a loop? So if when `cursor===0` will result in `cursor=liveSearchResults.length - 1`

File: src/components/common/Search/Search.tsx (hunk 1 above)
Comment (follan) [reply src/components/common/Search/Search.tsx]: Fixed

File: src/components/common/Search/Search.tsx (hunk 2 above)
Comment (follan) [reply src/components/common/Search/Search.tsx]: fixed

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-228.gearboxjs.preview.cogniteapp.com

**Pull request merged**